software/tracksight/backend/app/process/flask_apps/http_app.py

```diff
@@ -2,75 +2,75 @@
 Main REST component of the backend
 """
 
-from typing import Tuple
-from flask import Blueprint, jsonify, request, Response
+from flask import Blueprint, request
 
-from ..influx_handler import InfluxHandler as influx
-from ..influx_handler import NoDataForQueryException
+from .. import influx_handler as influx
 
 # HTTP processes for data that is not live
 app = Blueprint("http_app", __name__)
 
 
-def responsify(data):
-    """
-    Manual CORS?????? very bad idea
-    :param data: Data to JSON-ify.
-    :returns JSON-ified data response.
-    """
-    response = jsonify(data)
-    response.headers.add("Access-Control-Allow-Origin", "*")
-    return response
-
-
 @app.route("/")
-def hello_world() -> str:
+def hello_world():
```

Comment:
smh why'd you delete my types

Reply:
merge conflict :( also they don't return responses, they return dictionaries now